bindings/requests.go

```diff
@@ -5,8 +5,20 @@
 
 package bindings
 
-// WriteRequest is the object given to an dapr output binding
-type WriteRequest struct {
-	Data     []byte            `json:"data"`
-	Metadata map[string]string `json:"metadata"`
+// InvokeRequest is the object given to a dapr output binding
+type InvokeRequest struct {
+	Data      []byte            `json:"data"`
+	Metadata  map[string]string `json:"metadata"`
+	Operation string            `json:"operation"`
 }
+
+// OperationType defines an output binding operation
+type OperationType string
+
+// Non exhaustive list of operations. A binding can add operations that are not in this list.
+const (
+	GetOperation    = "get"
```

Comment:
I think we need to specify the exact type for strict typing..

e.g.
```
GetOperation OperationType = "get"
CreateOperation OperationType = "create"
...
```

Reply:
Changed